### DISCUSSION TITLE
Issue in Intro-to-Github-flow course

### DISCUSSION THREAD

**Discussion by mohdaffan3** [community/community#106720] [GitHub Education]
### Hi everyone,

,I started the Intro to Github flow course, and i completed it with ease but I think I messed up some settings in the repo created for me by Github for this course and now even though I completed all the requirements for the course, my assignments are getting rejected, is there anyway where I can start the course all over with new repo.

**Comment by subidit** [comment]
You can delete the current repo and then begin the course again.

**Reply by mohdaffan3** [reply]
Not getting the option to delete the repo, guess I have not been granted those permissions

**Reply by subidit** [reply]
when you click the start course button it will ask you to fork that repo into your own profile. Go ahead and do that and then follow instrustructions.
